main/ertp/ERTP-Introduction.md

```diff
@@ -0,0 +1,117 @@
+
+## Creating assets with ERTP
+
+In ERTP, *mints* create digital *assets*. Access to an asset type's
+mint lets you create more digital assets of that type. You can then
+store new assets in a *payment* or a *purse*. 
+- **Payments**: Assets you intend to trade with someone else.
+- **Purses**: Store assets until you use them for a trade.
+
+To send assets in ERTP:
+1. Withdraw them from a purse.
+2. Put them in a payment.
+3. Send them to a recipient object as a message.
+
+To receive assets in ERTP:
+1. Create a purse for the asset type you'll receive.
+2. Get access to the asset type you'll receive. (**Note:** you
+do not need access to the type's mint, just the specifc asset you'll 
+receive. 
+3. Receive the message with the payment and put the payment in
+your purse.
+
+## Security properties
+
+ERTP purse objects have a `deposit` message which takes a payment
+object as its argument. It first checks that the payment object is
+genuine and the same assay type as the purse takes (after all,
+quatloos are a very unstable asset, so you don't want to swap your
+very stable simoleans for them). . 
+
+If everything passes the checks, the asset moves from the payment to
+the purse. If there's a problem, it throws an error.
+
+After a successful deposit, ERTP guarantees:
+- The payment object is empty.
+- The purse contains the payment's full content.
+
+When the `deposit` call throws an error (i.e. something went wrong),
+ERTP guarantees: 
+- The alleged payment is in the same state as before the call.
+- The purse is in the same state as before the call.
+
+## Issuers and mints
+
+Other key ERTP objects are:
+
+- **Mints**: Issue new digital assets as a new Payment. Mints only
+issue one kind of asset (quatloos, simoleons, moola, etc.). We refer to
+that kind as the mint's *Brand*. So if a mint issues quatloos, it's a
+quatloo brand mint.  Only mints can issue new digital assets. To mint
+new assets of a particular type, you much have a reference to that
+type's mint 
+
+- **Issuers**: Creates empty purses and payments and maps minted
+ assets to them when the assets are added or removed. 
+ Issuers verify and move digital assets.
+
+An issuer's special admin facet is a Mint, and that Mint and Issuer are
+ associated with each other. With a reference to an issuer, you can
+ check the validity of a payment in that issuer's mint's assets, as
+ well as claim it either as a new payment to yourself or a purse you
+ control. 
+
+Issuers (i.e. their references) should be gotten from a trusted source
+and then relied upon as the decider of whether an untrusted payment is
+valid  
+
+**Note**: There is a one-to-one correspondence between a brand and a
+  mint, a mint and an issuer, and an issuer and a brand. In other
+  words, each mint issues a unique kind of digital asset, say,
+  quatloos, and only that kind of asset.
+
+## Amounts
+
+*Amounts* describe digital assets. Anyone can make one, and they can
```

Comment:
We should say something here about how an `amount` describes a digital asset, but there is no value in an amount. It's only a description, like a description of what is up for auction, or a bank account's balance.